internal/bminventory/inventory.go

```diff
@@ -6663,3 +6663,32 @@ func (b *bareMetalInventory) HandleVerifyVipsResponse(ctx context.Context, host
 	}
 	return b.clusterApi.HandleVerifyVipsResponse(ctx, *host.ClusterID, stepReply)
 }
+
+func getDefaultHighAvailabilityAndMasterCountParams(highAvailabilityMode *string, controlPlaneCount *int64) (*string, *int64) {
```

Comment:
maybe we can move it to common? File this huge already

Reply:
My thought was keep it here because this is the only place it is used